rest_api/controller/search.py

```diff
@@ -112,7 +112,6 @@ def _format_filters(filters):
                     f"Request with deprecated filter format ('{key}: null'). "
                     f"Remove null values from filters to be compliant with future versions"
                 )
-                continue
```

Comment:
Leaving this one out seems to change the logic as in line 123 the values (None here) will be added to new_filters dict.

Reply:
Good catch!!